File: acmp.ru/[0201] - [0300]/[0293] Налоги.py
n = int(input())
V = [int(i) for i in input().split()]
p = [int(i) for i in input().split()]
imax = 0

for i in range(1, n):
    if V[i] * p[i] > V[imax] * p[imax]:
        imax = i

print(imax + 1)

File: acmp.ru/[0201] - [0300]/[0293] Налоги.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;

    int imax = 0, V[n], p[n];

    for (int i = 0; i < n; i++)
        cin >> V[i];

    for (int i = 0; i < n; i++)
        cin >> p[i];


    for (int i = 0; i < n; i++)
        if (V[i] * p[i] > V[imax] * p[imax])
            imax = i;

    cout << imax + 1 << endl;

    return 0;
}
